[TOOLS-4762] Manual of API update (#110)
CUBRID/cubrid-manager-server: docs/api/paramdump.md

| compat_mode | |
| compat_numeric_division_scale | specifies how many decimal places to display in division results: no uses 9 places, yes depends on operand precision |
| compat_primary_key | |
| connection_timeout | |
| create_table_reuseoid | Determines which option should be used between `REUSE_OID` and `DONT_REUSE_OID` when table option is omitted in table creation |
| csql_auto_commit | |
| csql_history_num | number of SQL clauses stored in history log of CSQL interpreter |
| csql_single_line_mode | |
| cte_max_recursions | maximum number of iterations allowed for executing recursive parts of a CTE (Common Table Expression) query |
| cubrid_port_id | port of master process |
| data_aout_ratio | |
| data_buffer_neighbor_flush_nondirty | Determines whether to flush clean pages adjacent to dirty pages when victim candidate pages are flushed while the neighbor flush option is enabled  |
| data_buffer_neighbor_flush_pages | Number of adjacent pages which would be flushed using background flush |
| data_buffer_pages | Size of data buffer pages which is cached in memory by database server. Used in systems with version prior to 2008 R4.0 |

> | data_buffer_size | Ssize of data buffer which is cached in memory by database server |

Reviewer: ```suggestion
| data_buffer_size | Size of data buffer which is cached in memory by database server |
```

| data_file_os_advise | Parameter to enhance I/O performance which is applicable only to Unix-based system  |
| db_hosts | Determines the list of available database server hosts and the order of connection |
| db_volume_size | default size of the database volume created when the --db-volume-size option is omitted in the cubrid createdb and cubrid addvoldb utilities.  |

> | dblink_auto_commit | Dtermines wheter dblink query to be automatically committed immediately |

Reviewer: ```suggestion
| dblink_auto_commit | Determines whether dblink query to be automatically committed immediately |
```

| ddl_audit_log | Determines whether to log every `DDL` operation |

> | ddl_audit_log_size | Maximum size of the `DDLAUDIT' log file |

Reviewer: ```suggestion
| ddl_audit_log_size | Maximum size of the `DDLAUDIT` log file |
```

| log_volume_size | Default size of the log volume file created when the --log-volume-size is ommitted in the `cubrid createdb` utility |
| lru_buffer_ratio | |
| lru_hot_ratio | |
| max_agg_hash_size | Maximum memory size allocated per transaction for hashing tuple groups in queries that include aggregation |
| max_clients | Maximum number of clients allowed to connect to database server |
| max_filter_pred_cache_entries | maximum number of filtered index expression that can be cached in memory |
| max_flush_pages_per_second | Maximum flush capacity for the flush operation from buffer to disk. This parameter will be deprecated. |
| max_flush_size_per_second | Maximum flush capacity for the flush operation from buffer to disk. |
| max_hash_list_scan_size | Maximum memory size allocated per transaction for building hash table in queries that include subquery |
| max_plan_cache_clones | Maximum clone cache value that one plan can have |
| max_plan_cache_entries | Maxinum number of query execution plan cached in memory |
| max_query_cache_entries | Maximum number of queries that can be cached |

> | max_query_per_tran | Maximum nubmer of unclosed query that single transaction can hold at once  |

Reviewer: ```suggestion
| max_query_per_tran | Maximum number of unclosed query that single transaction can hold at once  |
```
